presto-druid/src/test/java/io/prestosql/plugin/druid/TestingDruidServer.java

```diff
@@ -17,7 +17,6 @@
 import com.google.common.io.Closer;
 import com.google.common.io.Resources;
 import io.prestosql.testing.assertions.Assert;
-import io.prestosql.tests.product.launcher.testcontainers.SelectedPortWaitStrategy;
```

Comment:
you probably need to remove pom.xml dep now

Reply:
Sorry, missed it.